src/doc/src/commands/cargo-uninstall.md

```diff
@@ -27,7 +27,7 @@ The installation root is determined, in order of precedence:
 
 ## OPTIONS
 
-### Install Options
+### Uninstall Options
```

Comment:
Thank you for your contribution!

This markdown file is generated. You'll need to update the manpage template instead: https://github.com/rust-lang/cargo/blob/82c489f1c612096c2dffc48a4091d21af4b8e046/src/doc/man/cargo-uninstall.md?plain=1#L24

See also the doc contribution guide: https://github.com/rust-lang/cargo/tree/master/src/doc#building-the-man-pages

Reply:
Thanks for the feedback! 

I moved the typo to the correct file.

Reply:
Thanks! Could you squash commits into one? We would like to keep every commit passing CI if possible.

Reply:
I will! Thanks for your patience!

Reply:
Hey @weihanglo, is there any way for me to stop the CI when I know that it's going to fail? It seems wasteful to run the full pipeline in that case.

Reply:
Push a new rev and the old one will get cancelled.

https://github.com/rust-lang/cargo/blob/82c489f1c612096c2dffc48a4091d21af4b8e046/.github/workflows/main.yml#L19-L21